pkg/controller/utils/datadog/metrics_forwarder.go

```diff
@@ -85,6 +100,9 @@ func newMetricsForwarder(k8sClient client.Client, namespacedName types.Namespace
 		sendMetricsInterval: defaultSendMetricsInterval,
 		metricsPrefix:       defaultMetricsNamespace,
 		stopChan:            make(chan struct{}),
+		errorChan:           make(chan error, 100),
```

Comment:
should we reduce this capacity?

Reply:
what is the rationale for this value?

Reply:
I couldn't find a rational quantification of this really, it depends on the errors and if the reconcile returns a `RequeueAfter` - I thought buffering 100 errors is enough in the worst case (when the reconcile loop keeps returning non nil errors continuously)